models/order.js

```diff
@@ -1,13 +1,19 @@
 'use strict';
 module.exports = (sequelize, DataTypes) => {
   var Order = sequelize.define('Order', {
-    customer_id: DataTypes.INTEGER,
-    payment_type: DataTypes.INTEGER
+    user_id: DataTypes.INTEGER,
+    payment_type_id: DataTypes.INTEGER
```

Comment:
Since you have it below in the associations, you shouldn't need to specify the payment_type_id here, I believe

Reply:
Good point 🙈 Will update this. Thanks!